diff --git a/s-chat-support_old.js b/s-chat-support.js
index 5f4c6de..1b5a9fb 100644
--- a/s-chat-support_old.js
+++ b/s-chat-support.js
@@ -138,7 +138,7 @@
             // ddp init
             var ddp = new sChatDDPClient(connectionOptions);
             // random userSessionId generator
-            var userSessionId = sChatRandom.id();
+            var userSessionId = sessionStorage.getItem('sChatUserSessionId') || sChatRandom.id();
             var submitMsg = function (e) {
                 var input = sChatBox().submitInput;
                 var key = e.keyCode || e.which;
@@ -195,6 +195,10 @@
                 console.log('SimpleChat.Support: Chat box connected!');
                 // create main chat box and append at the end of the 'body' tag
                 sChatBoxCreate();
+                // set session storage if don't exists
+                if (sessionStorage && !sessionStorage.getItem('sChatUserSessionId')) {
+                    sessionStorage.setItem('sChatUserSessionId', userSessionId);
+                }
                 // subscribe to the 'chat' publication
                 ddp.sub('Chat.messagesList', [clientAppId, userSessionId]);
                 ddp.sub('Meteor.users.adminStatus', [clientAppId]);
